packages/squidex/schema/crn/schemas/users.json

```diff
@@ -243,6 +243,41 @@
               "isRequiredOnPublish": false,
               "isHalfWidth": true
             }
+          },
+          {
+            "name": "status",
+            "isHidden": false,
+            "isLocked": false,
+            "isDisabled": false,
+            "properties": {
+              "fieldType": "String",
+              "defaultValue": "Active",
+              "allowedValues": ["Active", "Inactive"],
+              "isUnique": false,
+              "isEmbeddable": false,
+              "inlineEditable": true,
+              "createEnum": false,
+              "contentType": "Unspecified",
+              "editor": "Dropdown",
+              "label": "Status",
+              "isRequired": false,
+              "isRequiredOnPublish": false,
+              "isHalfWidth": true
+            }
+          },
+          {
+            "name": "inactiveSinceDate",
```

Comment:
i wonder if we need both of these fields at the same time?

Let me ask you this - can a user be inactive but not have an `inactiveSinceDate` set? Cause with your current proposal that would be possible, and your types seem to reflect it:
https://github.com/yldio/asap-hub/blob/0eb3e0b6dbf0bcc8d4a3c7e3d174a3d272cd3b19/packages/model/src/user.ts#L109-L110

What is even more strange is that the ticket does not seem to require such field, nor do any of the FE components seem to use it. 

If we know we are going to need such date then I suggest we make sure the requirements are clear. To me - if a user is set to inactive a date will be required, in which case I would be tempted to remove the `status` field and just use the `inactiveSinceDate`. You could leave both of them in the Response type so it's easier for FE to check the status, but the data provider would only return this one.